<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLineEdit" name="linePath"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnCd">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>cd</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QSplitter" name="splitter_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QPushButton" name="btnBack">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&lt;-</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Left</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnForward">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-&gt;</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Right</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnUp">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Up</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Up</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnHome">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Home</string>
                  </property>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="tableWidget">
                 <property name="maximumSize">
                  <size>
                   <width>560</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <column>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QPushButton" name="btnAdd">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>-&gt;</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnRemove">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;-</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btmClear">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>clear</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QTableWidget" name="tableWidget_2">
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <column>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Total:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineTotal">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Path on MyDisk:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineMyDisk"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnPath">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>31</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>srch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUpload">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>upload</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLineEdit" name="linePath_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnCd_2">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>cd</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QSplitter" name="splitter_3">
                 <property name="maximumSize">
                  <size>
                   <width>572</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QPushButton" name="btnBack2">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>&lt;-</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Left</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnForward2">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>-&gt;</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Right</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnUpdateDir">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Update</string>
                  </property>
                  <property name="shortcut">
                   <string>Alt+Up</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="btnHome2">
                  <property name="maximumSize">
                   <size>
                    <width>140</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add Dir</string>
                  </property>
                 </widget>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="tableWidget_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>560</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <widget class="QPushButton" name="btnAdd2">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>-&gt;</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btnRemove2">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>&lt;-</string>
                </property>
               </widget>
               <widget class="QPushButton" name="btmClear2">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>clear</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QTableWidget" name="tableWidget_4">
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>true</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Type</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>Size</string>
                  </property>
                 </column>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Total:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineTotal_2">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Path on MyDisk:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineMyDisk_6"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnPath2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>31</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>srch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btnUpload2">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string>upload</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
       <widget class="QPushButton" name="btnAddCoins">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>160</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add coins</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnUodateJson">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>101</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh Json</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>90</y>
          <width>781</width>
          <height>501</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnGetCoins">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>250</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>GetCoins</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>240</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>340</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>390</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnAddDir">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>380</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Add </string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDelteDir">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>460</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>DeleteDir</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>460</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMyDisk">
    <property name="title">
     <string>MyDisk</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionUsername_some_thing"/>
    <addaction name="actionUser_Options"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionAuthorize"/>
    <addaction name="actionChange_User"/>
   </widget>
   <addaction name="menuMyDisk"/>
   <addaction name="menuUser"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUsername_some_thing">
   <property name="text">
    <string>Username@some.thing</string>
   </property>
  </action>
  <action name="actionUser_Options">
   <property name="text">
    <string>User Options</string>
   </property>
  </action>
  <action name="actionAuthorize">
   <property name="text">
    <string>Authorize</string>
   </property>
  </action>
  <action name="actionChange_User">
   <property name="text">
    <string>Change User</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
